### PR TITLE
Minor updates in SDK docs 4.0.0 - Closes #782

### LINKED ISSUE #782: Minor updates in SDK docs 4.0.0
Minor updates are needed in the SDK docs v 4.0.0

### PR DESCRIPTION
**Pull request by chris529**
Closes #782

### CODE REVIEW

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 1]
```diff
@@ -274,8 +274,8 @@ image:launch-3_nodes.png[3 nodes diagram]
 To join the network as a new delegate, follow the steps below:
 
 . Create your own private account on the network.
-.. xref:{url_commander_commands_createaccount}[Generate the account credentials]
-.. Send some funds (which must be an adequate amount in order to register as a delegate), to the newly generated address.
+.. xref:{url_commander_commands_createaccount}[Generate the account credentials].
+.. Send some funds, (which must be an adequate amount in order to register as a delegate) to the newly generated address.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/launch.adoc]: I would remove the comma here
```suggestion
.. Send some funds (which must be an adequate amount in order to register as a delegate) to the newly generated address.
```

File: modules/ROOT/pages/references/changelog.adoc [hunk 2]
```diff
@@ -65,12 +65,12 @@ The array may be empty.
 . The `asset` property needs to contain the property `optionalKeys`.
 The value of this property is an array of pairwise distinct public keys.
 This array must be ordered lexicographically.
-Once the account is registered as a multisignature account, every outgoing transaction requires some signatures for some public keys in `optionalKeys` (the number of required signatures depends on the `numberOfSignatures` property and may also be zero).
+Once the account is registered as a multisignature account, every outgoing transaction requires some signatures for some public keys in `optionalKeys`, (the number of required signatures depends on the `numberOfSignatures` property and may also be zero).
```
Comment (Tschakki) [inline modules/ROOT/pages/references/changelog.adoc]: Please don't put a comma in front of every parentheses, if it wouldn't be there without the parentheses. Also, please check for other occurrences, where you added the comma.

File: modules/ROOT/pages/references/lisk-elements/api-client.adoc [hunk 3]
```diff
@@ -37,11 +37,12 @@ npm update @liskhq/lisk-api-client
...
-****
-A constructor is exposed, which takes an array of nodes and optionally an options object.
+
+A constructor is exposed which takes an array of nodes and optionally an options object.
 For convenience, helper functions are provided for creating API clients on specific networks with a default set of nodes.
 It is recommended to use these functions unless you are operating your own nodes, or there is a preference to use nodes provided by a third party.
 
+****
 NOTE: If the generic constructor is used, it is the user's responsibility to ensure that the specified nodes are on the correct network.
 ****
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/api-client.adoc]: Not sure what the stars are doing here. Please remove.

File: modules/ROOT/pages/references/lisk-elements/api-client.adoc [hunk 4]
```diff
@@ -148,7 +149,7 @@ Documentation for each resource can be found on the following pages:
 
 This initialises the client instance with an array of nodes and an optional configuration object.
 This is called in the constructor, and can be called again later if necessary.
-(Note that in practice it is usually easier just to create a new instance.)
+(Note that in practice it is usually easier just to create a new instance).
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/api-client.adoc]:
```suggestion
Note that in practice it is usually easier just to create a new instance.
```

File: modules/ROOT/pages/references/lisk-elements/transactions.adoc [hunk 5]
```diff
@@ -98,7 +98,7 @@ TransferTransaction(options)
 * `asset.amount` (required): The amount to transfer, (as a `string` in Beddows, the lowest denomination possible).
 * `recipientId` (required): The address of the recipient.
 * `data` (optional): Data to include in the transaction asset.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/transactions.adoc]:
```suggestion
* `data` (optional): Data to include in the transaction asset (must be a UTF8-encoded string of maximum 64 characters).
```

File: modules/ROOT/pages/references/lisk-elements/transactions.adoc [hunk 6]
```diff
@@ -98,7 +98,7 @@ TransferTransaction(options)
...
-(Must be a UTF8-encoded string of maximum 64 characters.)
+(Must be a UTF8-encoded string of maximum 64 characters).
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/transactions.adoc]: Remove this line

File: modules/ROOT/pages/tutorials/supply-chain/part1.adoc [hunk 7]
```diff
@@ -10,7 +10,7 @@
 :url_pi_ssh_over_usb: https://desertbot.io/blog/ssh-into-pi-zero-over-usb
 
 :url_setup: setup.adoc
-:url_transport_architecture: tutorials/supply-chain/part2.adoc#architecture
+:url_transport_architecture: tutorials/supply-chain/part3.adoc#architecture
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/supply-chain/part1.adoc]: I am not sure this part needs to be updated.

File: modules/ROOT/pages/tutorials/supply-chain/part3.adoc [hunk 8]
```diff
@@ -21,10 +21,10 @@
 :url_register_packet: http://localhost:3000/post-register-packet
 :url_start_transport: http://localhost:3000/post-start-transport
 
-:url_tutorials_transport_1_client: tutorials/supply-chain/part2.adoc#client
+:url_tutorials_transport_1_client: tutorials/supply-chain/part3.adoc#client
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/supply-chain/part3.adoc]: Not sure this part needs to be updated.

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good, just a few suggestions.

File: modules/ROOT/pages/references/changelog.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/references/changelog.adoc]: OK, now done 👍

File: modules/ROOT/pages/references/lisk-elements/api-client.adoc (hunk 3 above)
Comment (chris529) [reply modules/ROOT/pages/references/lisk-elements/api-client.adoc]: OK, done 👍

File: modules/ROOT/pages/references/lisk-elements/transactions.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/references/lisk-elements/transactions.adoc]: Ok, done 👍

File: modules/ROOT/pages/tutorials/supply-chain/part1.adoc (hunk 7 above)
Comment (chris529) [reply modules/ROOT/pages/tutorials/supply-chain/part1.adoc]: OK, I have reverted it 👍

File: modules/ROOT/pages/tutorials/supply-chain/part3.adoc (hunk 8 above)
Comment (chris529) [reply modules/ROOT/pages/tutorials/supply-chain/part3.adoc]: Yes, I think you are right.I have also reverted it 👍

File: modules/ROOT/pages/references/lisk-elements/transactions.adoc (hunk 5 above)
Comment (Tschakki) [reply modules/ROOT/pages/references/lisk-elements/transactions.adoc]: Why did you mark it at resolved? It isn't applied yet.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/references/lisk-elements/transactions.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/pages/references/lisk-elements/transactions.adoc]: Oh, Sorry, I thought I had already removed it.

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**